src/main/java/ludogorie_soft/reservations_platform_api/GEH/APIException.java

```diff
@@ -0,0 +1,17 @@
+package ludogorie_soft.reservations_platform_api.GEH;
```

Comment:
We already have a package `exception` - it is better to move all these classes there

Reply:
made a correction to the package name